### ISSUE TITLE
incomplete ontology sync

### ISSUE THREAD

**Issue by xristy** [buda-base/lds-pdi#131]
I added `ontology/adm/types/git_repos.ttl` to owl-schema and updated the `ont-policy.rdf` accordingly.

However, visiting [GitRepo](http://purl.bdrc.io/ontology/adm/types/GitRepo/) is not returning the the GitRepo ontology and attempting to load the `admin.ttl` fails on importing `GitRepo`.

**Comment by MarcAgate** [comment]
It is working now and I don't know what happened (could it be a cache issue?). Couldn't find anything in the logs... see http://purl.bdrc.io/ontology/adm/types/GitRepo/ or http://purl.bdrc.io/ontology/admin/GitRepo and http://purl.bdrc.io/ontology/admin.ttl

**Comment by xristy** [comment]
Thanks for getting it working. Three comments:

1. Individuals such as GR0003 under GitRepo class should be viewable rather than the current stub like display.

2. Having discussed the two distinct URLs for GitRepo: ontology/adm/types/GitRepo vs ontology/admin/GitRepo, I understand that I used a seemingly poor choice for the ontology baseURI in git_repos.ttl and all the other factored out *.ttl. However, I'm not wanting to gratuitously change the baseURI to ontology/admin/GitRepo/ which will mess with the lds-pdi. So this remains a note that may be revisited if other unexpected results arise in the future.

3. There's also a question for me as to how to know what semantics to attach to the trailing `/` vs none.

**Comment by MarcAgate** [comment]
Regarding point 3: there is no semantics attached to the trailing / http://purl.bdrc.io/ontology/adm/types/GitRepo/ is working because this URI is recognized as a valid BaseURI, when http://purl.bdrc.io/ontology/adm/types/GitRepo is not, according to BaseURI definitions found in any ontology file loaded through the FileManager and OntPolicy.rdf file.

So either we leave it as is or we decide that both URIs are BaseURI, regardless the trailer /. Ignoring the trailer / would require to tweak ldspdi code.

I don't think the current BaseURIs are poor choices. No change is required from my point of view as we have one BaseURI pointing to the ontology view and another one, that conforms to our prefixes convention (ontology/admin/GitRepo) that points to a owl class view.

I am going to check what it takes to get a useful view of all these individuals (these might need to be displayed using a specific logic as they are instances of classes, instead of classes).

**Comment by xristy** [comment]
Thanks.

If we leave the `/` _as is_ then I guess there should be a note somewhere to use URLs exactly as written w/ or w/o the `/`.

I'm happy to leave the baseURIs alone. I recall that I ended up assigning them the way I did because of validation issues when I doing the refactoring.

Yes the correct resource URI's are like [http://purl.bdrc.io/admindata/GR0002.ttl](http://purl.bdrc.io/admindata/GR0002.ttl).
